contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -365,12 +374,11 @@ abstract contract PoSValidatorManager is
         uint64 weight = valueToWeight(lockedValue);
         bytes32 validationID = _initializeValidatorRegistration(registrationInput, weight);
 
-        $._posValidatorInfo[validationID] = PoSValidatorInfo({
-            owner: _msgSender(),
-            delegationFeeBips: delegationFeeBips,
-            minStakeDuration: minStakeDuration,
-            uptimeSeconds: 0
-        });
+        // Efficient Assignment
```

Comment:
I don't think these comments are necessary, unless this is going against some established Solidity convention that would be unexpected to see.